guides/v2.2/architecture/archi_perspectives/components/modules/mod_depend.md

```diff
@@ -31,13 +31,45 @@ A module with a *hard dependency* on another module cannot function without the
 * Deserializes an object declared in another module.
 * Uses or modifies the database tables used by another module.
 
+Module hard dependency defines in the `require` section of `app/code/<Vendor>/<Module>/composer.json` file. For example, :
+
+```json
+  ...
+  "require": {
+        "magento/module-catalog": "103.0.*",
+        "magento/module-email": "101.0.*",
+        "magento/module-media-storage": "100.3.*",
+        "magento/module-store": "101.0.*",
+        "magento/module-theme": "101.0.*",
+        "magento/module-ui": "101.1.*",
+        "magento/module-variable": "100.3.*",
+        "magento/module-widget": "101.1.*",
+        "magento/module-authorization": "100.3.*"
+   },
+   ...
+```
+
 ### Soft dependencies
 
 A module with a *soft dependency* on another module can function properly without the other module, even if it has a dependency on the other module. These modules:
 
 * Directly check another module's availability.
 * Extend another module's configuration.
-* Extend another module's {% glossarytooltip 73ab5daa-5857-4039-97df-11269b626134 %}layout{% endglossarytooltip %}.
+* Extend another module's [layout](https://glossary.magento.com/layout).
+
+Module hard dependency defines in the `<sequence>` section of `app/code/<Vendor>/<Module>/etc/module.xml` file. For example, :
```

Comment:
Please adjust this part to make it more clear
```suggestion
Module soft dependency is defined in the `<sequence>` section of `app/code/<Vendor>/<Module>/etc/module.xml` file. For example, :
```

We are talking about soft dependency here, right? :)

Reply:
Golden eye, @rogyar 👍 

Many thanks